optaplanner-core/src/main/java/org/optaplanner/core/api/score/holder/ScoreHolder.java

```diff
@@ -21,7 +21,7 @@
 import org.optaplanner.core.api.score.Score;
 
 /**
- * This is the base interface for all score holder implementations, used exclusively by Drools-based score calculators.
+ * This is the base interface for all score holder implementations.
```

Comment:
All changes you see to `ScoreHolder` in this PR are simplifications now that CS-D is not using it anymore.
(Specifically, the removal of `Object... justifications` method overloads. That was *only used by CS-D*.)